aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js

```diff
@@ -69,62 +70,45 @@ ContainerDrivenLayout.propTypes = {
   containerRef: PropTypes.object,
 };
 
-const ContentLayout = ({ containerRef }) => {
-  // containerRef is for demonstration purposes on this site. Most
-  // implementations should likely remove.
-  const breakpoint = useContext(ResponsiveContext);
-  const [filteredApps, setFilteredApps] = useState(allApps);
-  const [filters, setFilters] = useState({});
-
-  // apply filters
-  useEffect(() => {
-    let results = [];
-    let numFilters = 0;
-    Object.values(filters).forEach(filter => (numFilters += filter.length));
-
-    if (numFilters === 0) {
-      results = allApps;
-    } else {
-      results = allApps.filter(app => {
-        let result = true;
-        Object.keys(filters).forEach(key => {
-          if (filters[key].length > 0) {
-            const match = filters[key].some(item => app[key].includes(item));
-            if (match === false) {
-              result = false;
-            }
-          }
-        });
-        return result;
-      });
-    }
+const properties = {
+  categories: {
+    label: 'Categories',
+    options: [
+      { label: 'Machine Learning', value: 'Machine learning' },
+      { label: 'Big Data', value: 'Big Data' },
+      { label: 'Data Analytics', value: 'Data analytics' },
+      { label: 'Developer Tools', value: 'Developer tools' },
+      { label: 'Monitoring', value: 'Monitoring' },
+      { label: 'Database', value: 'Database' },
+      { label: 'Data Protection', value: 'Data Protection' },
+    ],
+  },
+  publisher: {
+    label: 'Publishers',
+  },
+  delivery: { label: 'Delivery Methods' },
+  pricing: { label: 'Pricing Models' },
+};
 
-    setFilteredApps(
-      results.reduce((acc, cur) => {
-        if (!acc.includes(cur)) {
-          acc.push(cur);
-        }
-        return acc;
-      }, []),
-    );
-  }, [filters, setFilteredApps]);
+const ContentLayout = () => {
+  const breakpoint = useContext(ResponsiveContext);
+  const showFiltersButton = breakpoint === 'small' || breakpoint === 'xsmall';
+  const sidebar = <DataFilters layer={showFiltersButton ? true : undefined} />;
```

Comment:
I think we are waiting for https://github.com/grommet/grommet/pull/6740 to be merged before this PR is complete, correct?